src/main/java/com/example/kanbanserv/domain/error/ErrorDetails.java

```diff
@@ -0,0 +1,31 @@
+package com.example.kanbanserv.domain.error;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ErrorDetails {
+
+    @JsonProperty("field")
```

Comment:
Do we need to specify the argument in `@JsonProperty` even when the variable name matches with the Json field name? 
Check once whether writing only `@JsonProperty` instead of `@JsonProperty("field")` works or not.

Reply:
Yes, it works. Thank you!